src/test/java/org/wiremock/integrations/testcontainers/WireMockContainerJUnit5Test.java

```diff
@@ -1,55 +1,83 @@
-          "hello-world-resource-response.xml");
-
-
-  @ParameterizedTest
-  @ValueSource(strings = {
-          "hello",
-          "/hello"
-  })
-  public void helloWorld(String path) throws Exception {
-    // given
-    String url = wiremockServer.getUrl(path);
-
-    // when
-    HttpResponse<String> response = TestHttpClient.newInstance().get(url);
-
-    // then
-    assertThat(response.body())
-        .as("Wrong response body")
-        .contains("Hello, world!");
-  }
-
-  @Test
-  public void helloWorldFromFile() throws Exception {
-    // given
-    String url = wiremockServer.getUrl("/hello-from-file");
-
-    // when
-    HttpResponse<String> response = TestHttpClient.newInstance().get(url);
-
-    // then
-    assertThat(response.body())
-        .as("Wrong response body")
-        .contains("Hello, world!");
-  }
+    @Container
+    public WireMockContainer wiremockServer = new WireMockContainer("2.35.0")
+            .withMapping("hello", WireMockContainerTest.class, "hello-world.json")
+            .withMapping("hello-resource", WireMockContainerTest.class, "hello-world-resource.json")
+            .withFileFromResource("hello-world-resource-response.xml", WireMockContainerTest.class,
+                    "hello-world-resource-response.xml");
+
+
+    @ParameterizedTest
+    @ValueSource(strings = {
+            "hello",
+            "/hello"
+    })
+    public void helloWorld(String path) throws Exception {
+        // given
+        String url = wiremockServer.getUrl(path);
+
+        // when
+        HttpResponse<String> response = TestHttpClient.newInstance().get(url);
+
+        // then
+        assertThat(response.body())
+                .as("Wrong response body")
+                .contains("Hello, world!");
+    }
+
+    @Test
+    public void helloWorldFromFile() throws Exception {
+        // given
+        String url = wiremockServer.getUrl("/hello-from-file");
+
+        // when
+        HttpResponse<String> response = TestHttpClient.newInstance().get(url);
+
+        // then
+        assertThat(response.body())
+                .as("Wrong response body")
+                .contains("Hello, world!");
+    }
+
+    @Test
+    public void defaultBannerDisabled() {
+        WireMockContainer wireMockContainerSpy = Mockito.spy(new WireMockContainer("2.35.0"));
+
+        wireMockContainerSpy.configure();
+
+        verify(wireMockContainerSpy).withCliArg("--disable-banner");
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {
+            "--verbose",
+            " --verbose ",
+            "--verbose --help"
+    })
+    public void showBannerWhenStartingVerbose(String verboseArg) {
+        WireMockContainer wireMockContainerSpy = Mockito.spy(new WireMockContainer("2.35.0"));
+        wireMockContainerSpy.withCliArg(verboseArg);
+
+        wireMockContainerSpy.configure();
+
+        verify(wireMockContainerSpy, times(0)).withCliArg("--disable-banner");
+    }
```

Comment:
I am against using Mockito without a strong reason. In this case we can add a new API method that provides the necessary observability and might be useful elsewhere, e.g. `String toCLIArgs()`